### PR TITLE
Add schema internacionalization

### PR DESCRIPTION
**Pull request by vitoria**
#### What is the purpose of this pull request?
Add schema internationalization

#### What problem is this solving?
This component's schema did not have internationalization.

#### How should this be manually tested?
https://minicartinter--storecomponents.myvtex.com/

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/locales/pt-BR.json [hunk 1]
```diff
@@ -1,4 +1,9 @@
 {
+    "editor.minicart.title": "Sacola",
```
Comment (claudivanfilho) [inline react/locales/pt-BR.json]: muda isso pfr

**Review by claudivanfilho — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Schema internationalization
```
Comment (brunojdo) [inline CHANGELOG.md]: Add internationalization into minicart schema

**Review by brunojdo — CHANGES_REQUESTED** [review]

File: react/locales/pt-BR.json [hunk 3]
```diff
@@ -1,4 +1,9 @@
 {
+    "editor.minicart.title": "Carrinho de Compras",
+    "editor.minicart.description": "Um simples carrinho de compras que mostra todos os produtos adicionados a ele",
+    "editor.minicart.showRemoveButton.title": "Mostrar botão de remover produto",
+    "editor.minicart.labelMiniCartEmpty.title": "Texto que irá aparecer quando o carrinho estiver vazia",
```
Comment (andreldsa) [inline react/locales/pt-BR.json]: estiver vazio*

**Review by andreldsa — APPROVED** [review]
Fix then ship it!

**Review by brunojdo — APPROVED** [review]

**Review by claudivanfilho — APPROVED** [review]

**Pull request merged**